cmd/kubeadm/test/cmd/init_test.go

```diff
@@ -65,14 +67,19 @@ func TestCmdInitToken(t *testing.T) {
 			expected: false,
 		},
 		{
-			name:     "valid token is accepted",
-			args:     "--token=abcdef.0123456789abcdef",
-			expected: true,
+			name:          "valid token is accepted",
+			args:          "--token=abcdef.0123456789abcdef",
+			expected:      true,
+			skipOnWindows: true,
```

Comment:
do you have an explanation / logs why this test is failing on Windows for kubeadm?
`kubeadm init ... --token`  seems like a portable scenario across Windows/Linux, so might be a better idea to fix it instead of skipping.

Reply:
It seems that the kubeadm build process was a bit off. I've sent a PR for it: https://github.com/kubernetes-sigs/windows-testing/pull/357

@TheOneTheOnlyJJ, you can remove the kubeadm test skips, we should be able to merge the PR mentioned above soon.

@neolit123, I agree, we shouldn't skip any tests. However, the PRs [1][2][3] that are fixing the issues are still up for review for quite some time. We'd like to at least have a signal, to make sure regressions won't end up in the repository as before.

[1] https://github.com/kubernetes/kubernetes/pull/110566
[2] https://github.com/kubernetes/kubernetes/pull/114136
[3] https://github.com/kubernetes/kubernetes/pull/114607